packages/eslint-plugin-choreography-ts/tests/no-outside-choreographic-operator.test.ts

```diff
@@ -0,0 +1,270 @@
+import { RuleTester } from "@typescript-eslint/rule-tester";
+import noOutsideOperatorRule from "../src/no-outside-choreographic-operator";
+
+const ruleTester = new RuleTester({
```

Comment:
If you're going to define this in all of the test files, it might make sense to define it once in a separate file and import it from the test files.

Reply:
You mean importing things like the rule name and `RuleTester`? I'm not sure how that will help with succinctness.

Reply:
The only import that is the same between test files is `RuleTester`, so I'm not sure how much it will help putting all the imports in another separate file.

Reply:
@ImpregnableProgrammer  I was thinking of making `common.ts` inside `tests` with

```
export const ruleTester = new RuleTester({
  parser: "@typescript-eslint/parser",
  parserOptions: {
    project: "tsconfig.json",
    tsconfigRootDir: __dirname + "/fixtures", // This directiory and its files needed to test with type information
  },
});
```

I know it's just a couple of lines but we will have more rules and it doesn't feel right if all of the test files have the same setup.